src/plot_api/to_image.js

```diff
@@ -196,7 +196,7 @@ function toImage(gd, opts) {
     }
 
     return new Promise(function(resolve, reject) {
-        Registry.call('plot', [clonedGd, data, layoutImage, configImage])
+        plotApi.plot(clonedGd, data, layoutImage, configImage)
```

Comment:
just curious, why did you choose not to use `Registry` here?

Reply:
`Plotly.toImage` shouldn't be in a circular dep with the other `Plotly` methods. `Plotly.toImage` should call `Plotly.plot` but never the other way around.

Ideally, I think the api method registry should only be used in `components/` and in the subplot modules (for interaction updates). `Plotly` methods need to call component methods (e.g `.draw`) and some components need to call `Plotly` methods (e.g. updatemenus / mode bar button click) - this is circular, hence the need for an api method registry.